.github/workflows/kind.yaml

```diff
@@ -15,6 +15,7 @@ env:
   KIND_CONFIG: .github/kind-config.yaml
   TIMEOUT: 2m
   LOG_TIME: 30m
+  duration: 5
```

Comment:
Do we still need this change now that the duration is passed as a CLI flag to `cilium connectivity test --perf`?

Reply:
right. left overs.